R/setup.R

```diff
@@ -881,21 +881,24 @@ metadata_add_taxonomic_change <- function(dataset_id, find, replace, reason, tax
   set_name <- "taxonomic_updates"
   metadata <- read_metadata_dataset(dataset_id)
 
-  to_add <- list(find = find, replace = replace, reason = reason, taxonomic_resolution = taxonomic_resolution)
+  to_add <- dplyr::tibble(find = find, replace = replace, reason = reason, taxonomic_resolution = taxonomic_resolution)
 
   # Add `set_name` category if it doesn't yet exist
   if (all(is.na(metadata[[set_name]]))) {
-    metadata[[set_name]] <- list()
+    data <- to_add
```

Comment:
does Thi one need sorting too?

Reply:
yes! I thought we did that, but it must have been lost as I jumped branches. Redoing that now and will test.